src/GLTypes.jl

```diff
@@ -220,8 +225,17 @@ function GLVertexArray(bufferdict::Dict, program::GLProgram)
             )
             bind(buffer)
             attribLocation = get_attribute_location(program.id, attribute)
+            glslType = attributeTypes[Symbol(attribute)]
             (attribLocation == -1) && continue
-            glVertexAttribPointer(attribLocation, cardinality(buffer), julia2glenum(eltype(buffer)), GL_FALSE, 0, C_NULL)
+            if glslType in GLSLFloatTypes
```

Comment:
This differentiation seems unnecessary, since `glVertexAttribPointer` works for all types (but double) as far as i can tell by: https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glVertexAttribPointer.xhtml

Reply:
In that case, the if statement shouldn't be needed, unless you want to throw an error the the attribute type is compatible?

Reply:
We could do a check ;) but that will be a bit more complicated, because of colors etc!